test-utils/src/commonMain/kotlin/legacy.kt

```diff
@@ -106,17 +106,12 @@ abstract class KoinUIThreadViewModelTest<KVMC : KoinUIThreadViewModelTest.KoinVi
     }
 }
 
-@Deprecated("Moved to test-utils-base", ReplaceWith("assertFrozen(any, message, allowBoolean)", "com.splendo.kaluga.test.base.assertFrozen"))
-fun assertFrozen(any: Any, message: String? = null, allowBoolean: Boolean = false) = com.splendo.kaluga.test.base.assertFrozen(any, message, allowBoolean)
-@Deprecated("Moved to test-utils-base", ReplaceWith("assertNotFrozen(any, message, suppressWarning)", "com.splendo.kaluga.test.base.assertNotFrozen"))
-fun assertNotFrozen(any: Any, message: String? = null, suppressWarning: Boolean = false) = com.splendo.kaluga.test.base.assertNotFrozen(any, message, suppressWarning)
 @Deprecated("Moved to test-utils-base", ReplaceWith("awaitAllBlocking(deferreds)", "com.splendo.kaluga.test.base.awaitAllBlocking"))
 fun <T> awaitAllBlocking(vararg deferreds: Deferred<T>): List<T> = com.splendo.kaluga.test.base.awaitAllBlocking(*deferreds)
 @Deprecated("Moved to test-utils-base", ReplaceWith("captureFor(duration)", "com.splendo.kaluga.test.base.captureFor"))
 suspend fun <T> Flow<T>.captureFor(duration: Duration): List<T> = this.captureFor(duration)
-@Deprecated("Moved to test-utils-base", ReplaceWith("testBlockingAndCancelScope(context, freezeResult, block)", "com.splendo.kaluga.test.base.testBlockingAndCancelScope"))
+@Deprecated("Moved to test-utils-base", ReplaceWith("testBlockingAndCancelScope(context, block)", "com.splendo.kaluga.test.base.testBlockingAndCancelScope"))
```

Comment:
Maybe remove deprecated classes already? I guess they are technically not even released yet though.

Reply:
We'll deal with this in a separate PR if needed.